csfieldguide/chapters/content/en/complexity-and-tractability/sections/the-travelling-salesman-problem.md

```diff
@@ -2,25 +2,25 @@
 
-Researchers have spent a lot of time trying to find efficient solutions to the Travelling Salesman Problem, yet have been unable to find a *tractable* algorithm for solving it.
+Researchers have spent a lot of time trying to find efficient solutions to the travelling salesman problem, yet have been unable to find a *tractable* algorithm for solving it.
 As you learnt in the previous section, *intractable* algorithms are very slow, to the point of being impossible to use.
 As the only solutions to TSP are intractable, TSP is known as an *intractable problem*.
 
 It hasn’t actually been *proven* that there is no tractable solution to TSP, although many of the world’s top computer scientists have worked on this problem for the last 40 years, trying to find a solution but without success.
 What they have managed to do is find thousands of other problems that are also intractable, and more importantly, if a solution is found for any one of these problems, we know how to convert it to a solution for any of the others (these are called NP-complete problems).
 They all stand and fall together, including the TSP problem.
-So it's not just being lazy if you give up on finding an optimal TSP algorithm — people have tried for decades and not found a tractable algorithm.
-Of course, this is also a strong motivator to try to find one — if you do, you will have solved thousands of other problems at the same time!
+So it's not just being lazy if you give up on finding an optimal TSP algorithm &ndash; people have tried for decades and not found a tractable algorithm.
+Of course, this is also a strong motivator to try to find one &ndash; if you do, you will have solved thousands of other problems at the same time!
 This is a great thing for a researcher to do, but if you have a program to get finished by the end of the month, it's not a good bet to work on it.
 
 Current algorithms for finding the optimal TSP solution aren't a lot better than simply trying out all possible paths through the map (as in the interactive at the start of this chapter).
 The number of possible paths gets out of hand; it's an intractable approach.
 In the project below you'll be estimating how long it would take.
 
 While TSP was originally identified as being the problem that sales people face when driving to several different locations and wanting to visit them in the order that leads to the shortest route (less petrol usage), the same problem applies to many other situations as well.
-Courier and delivery companies have variants of this problem — often with extra constraints such as limits on how long a driver can work for, or allowing for left hand turns being faster than right-hand ones (in NZ at least!)
+Courier and delivery companies have variants of this problem &ndash; often with extra constraints such as limits on how long a driver can work for, or allowing for left hand turns being faster than right-hand ones (in NZ at least!)
```

Comment:
Need a full stop at the end of the sentence.
Some might not recognise 'NZ', maybe we could generalise it to be about left-side traffic instead - or change the example to be understood regardless of traffic direction